### DISCUSSION TITLE
Idea for output folders split per Google origin tag (webupload vs bakcup_and_sync)

### DISCUSSION THREAD

**Discussion by mjteebone** [TheLastGimbus/GooglePhotosTakeoutHelper#278] [I NEED HELP]
I got an idea that i found when struggling to clean my own photo library. The google json file holds a tag indicating the origin of particular photo - was it uploaded via phone app or as backup from google drive. In my use case, I have tons of duplicates because at some point I manually uploaded big folder with photos to google drive and they all got backed up to google photos (it used to work this way some years ago). Now trying to clean this mess it would be very helpful (at least to me) to have the option to split the  ALL_FILES folder into subfolders indicating the origin which is taken from the last json tag... Maybe someone else is also having this problem?

**Comment by TheLastGimbus** [comment]
Sorry but this seems like a very niche edge case that will be confusing for 99% of people

**Reply by TheLastGimbus** [reply]
You can try to solve this with some python yourself :eye:
